### PR TITLE
COMPAT/TST: Statespace: Standardized residuals: fix compatibility with Scipy 0.11, add tests

### PR DESCRIPTION
**Pull request by ChadFulton**
Scipy 0.11 didn't have the `check_finite` argument to `cho_factor`.

### CODE REVIEW

**Comment by ChadFulton** [comment]
Once I get this passing in Travis (hopefully first try gets it), hopefully we can merge this soon. Then the other recent PRs will pass, and hopefully we can work towards merging them.

**Comment by ChadFulton** [comment]
This is ready for merge. Should be easy to review (it's just a few lines of changes and a new test).

File: statsmodels/tsa/statespace/tests/test_kalman.py [hunk 1]
```diff
@@ -675,4 +676,34 @@ def test_filtered_state(self):
...
+    Just uses a different calculation method on a univariate series.
+    """
+
+    # Get the dataset
+    true = results_kalman_filter.uc_uni
+    data = pd.DataFrame(
+        true['data'],
+        index=pd.date_range('1947-01-01', '1995-07-01', freq='QS'),
+        columns=['GDP']
+    )
```
Comment (josef-pkt) [inline statsmodels/tsa/statespace/tests/test_kalman.py]: I don't really like the closing parenthesis in front. I would prefer it to be below the `c` of columns

**Comment by josef-pkt** [comment]
Looks fine to me. can be merged

Can you easily rebase and force push? not strictly necessary

**Comment by ChadFulton** [comment]
Rebased. Good to merge when green.

**Comment by josef-pkt** [comment]
merged, that was number 3

Thanks @ChadFulton

**Pull request merged**